Python/Product/PythonTools/PythonTools/DiagnosticsProvider.cs

```diff
@@ -69,7 +69,7 @@ internal sealed class DiagnosticsProvider {
 
         private static readonly Regex InterestingApplicationLogEntries = new Regex(
             @"^Application: (devenv\.exe|.+?Python.+?\.exe|ipy(64)?\.exe)",
-            RegexOptions.IgnoreCase | RegexOptions.CultureInvariant
+            RegexOptions.IgnoreCase | RegexOptions.CultureInvariant | RegexOptions.Compiled
```

Comment:
There's no benefit to compiling this regex - it isn't used nearly enough. We deliberately default to not using `Compiled` unless it can be proven to be worthwhile.

Reply:
There was a benefit while collecting full log. I haven't measured it but it was visually noticeable.